README.md

```diff
@@ -13,13 +13,13 @@ possible.
 
 ## How it works
 
-Turbinia is composed of different components for the client, server and the
-workers. These components can be run in the Cloud, on local machines, or as a
-hybrid of both. The Turbinia client makes requests to process evidence to the
-Turbinia server. The Turbinia server creates logical jobs from these incoming
-user requests, which creates and schedules forensic processing tasks to be run
+Turbinia is composed of different components for the client, server, and workers. 
+These components can be run in the Cloud, on local machines, or as a hybrid of both. 
```

Comment:
adding new lines has no impact on markdown as it will still concatenate all lines.